src/seedu/addressbook/commands/EditCommand.java

```diff
@@ -0,0 +1,75 @@
+package seedu.addressbook.commands;
+
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Removes a person identified using its last displayed index from the address book
+ * and adds a new person as the last entry in the address book.
```

Comment:
Too much details for the header comment, e.g. `removes` and `adds` are too detail.